network_simulator/main_event_loop.py

```diff
@@ -84,4 +85,7 @@ def run(self):
 
         logger.info("Finished running main Event loop.")
 
-        # TODO(team): Output final Statistics
```

Comment:
There is another TODO to output final Statistics a few lines above (when a warning occurs).